src/core/lib/security/credentials/tls/grpc_tls_certificate_distributor.cc

```diff
@@ -328,6 +328,7 @@ void grpc_tls_certificate_distributor::CancelTlsCertificatesWatch(
 /** -- Wrapper APIs declared in grpc_security.h -- **/
 
 grpc_tls_identity_pairs* grpc_tls_identity_pairs_create() {
+  grpc_core::ExecCtx exec_ctx;
```

Comment:
I think these three functions don't really need an `ExecCtx`, because there's absolutely nothing going on in these functions that could require any callbacks.